From 9f92f20d1d1f2e2e3e362a1981588e1f59d710d6 Mon Sep 17 00:00:00 2001
From: Wyon Bi <bivvy.bi@rock-chips.com>
Date: Fri, 9 Oct 2020 16:08:26 +0800
Subject: [PATCH] dt-bindings: phy: Document Rockchip Naneng eDP Transmitter
 PHY bindings

Add DT binding documentation for Naneng eDP Transmitter PHY IP used
in Rockchip's RK3568 SoC.

Change-Id: Id45165ccaef7d82f590e8d6ff26c6b6a0784314f
Signed-off-by: Wyon Bi <bivvy.bi@rock-chips.com>
---
 .../bindings/phy/phy-rockchip-naneng-edp.txt  | 26 +++++++++++++++++++
 1 file changed, 26 insertions(+)
 create mode 100644 Documentation/devicetree/bindings/phy/phy-rockchip-naneng-edp.txt

diff --git a/Documentation/devicetree/bindings/phy/phy-rockchip-naneng-edp.txt b/Documentation/devicetree/bindings/phy/phy-rockchip-naneng-edp.txt
new file mode 100644
index 000000000000..773dd8b244ed
--- /dev/null
+++ b/Documentation/devicetree/bindings/phy/phy-rockchip-naneng-edp.txt
@@ -0,0 +1,26 @@
+Rockchip Naneng eDP Transmitter PHY
+-----------------------
+
+Required properties:
+- compatible:	should be "rockchip,rk3568-edp-phy"
+- reg:		register range for the PHY.
+- clocks:	Must contain an entry in clock-names.
+		See ../clocks/clock-bindings.txt for details.
+- clock-names:	Must contain "refclk" and "pclk".
+- resets:	Must contain an entry for each in reset-names.
+		See ../reset/reset.txt for details.
+- reset-names:	Must contain "apb".
+- #phy-cells:	Must be 0
+
+Example:
+
+	edp_phy: edp-phy@fdcb0000 {
+		compatible = "rockchip,rk3568-edp-phy";
+		reg = <0x0 0xfdcb0000 0x0 0x8000>;
+		clocks = <&pmucru XIN_OSC0_EDPPHY_G>, <&cru PCLK_EDPPHY_GRF>;
+		clock-names = "refclk", "pclk";
+		resets = <&cru SRST_P_EDPPHY_GRF>;
+		reset-names = "apb";
+		#phy-cells = <0>;
+		status = "disabled";
+	};
-- 
2.35.3

